### ISSUE TITLE
SSHD service not installed

### ISSUE THREAD

**Issue by MichaelMc64** [MicrosoftDocs/windowsserverdocs#4924]
In a Powershell Administrator console, ran "Add-WindowsCapability -Online -Name OpenSSH.Server~~~~0.0.1.0"
This returned the information:
Path:
Online: True
RestartNeeded: False

However, Get-service sshd returns the error: 
get-service: Cannot find any service with service name 'sshd'.

One of us is missing a step. Since I'm following all the steps in the article, my guess is it's you.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 18a5a4e5-2604-23c2-1374-68a6974170bc
* Version Independent ID: f93809cc-eb5e-6b9b-ed58-c8811a8ee21f
* Content: [Installation of OpenSSH For Windows Server](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse)
* Content Source: [WindowsServerDocs/administration/OpenSSH/OpenSSH_Install_FirstUse.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/administration/OpenSSH/OpenSSH_Install_FirstUse.md)
* Product: **windows-server**
* Technology: **administration**
* GitHub Login: @maertendMSFT
* Microsoft Alias: **damaerte**

**Comment by illfated** [comment]
What is the result you get when you run the following PowerShell command (as Administrator)?

```powershell
Get-WindowsCapability -Online | ? Name -like 'OpenSSH*'
```

**Comment by illfated** [comment]
Also, I believe you forgot  to start the SSHD service manually:
```ruby
Start-Service sshd

Set-Service -Name sshd -StartupType 'Automatic'
```

**Comment by illfated** [comment]
Does it make any difference if you restart your server machine?

**Comment by deepyes02** [comment]
Hello I think in the insider preview this doesn't work with the latest update

**Comment by illfated** [comment]
Fair point. I hope this can be documented somehow, but I recommend searching for confirmation on this in the Microsoft forums and reference any news about that back here. The MS Docs server team does not have the capacity to follow up on all these issue tickets, unless they contain documentable changes needed in the referenced document (page).

Please feel free to comment again whenever you have got any useful leads on how this can be documented properly for future readers of the documentation.

**Comment by MichaelMc64** [comment]
I've no idea what changed, but this is now working. Before initial post I did try rebooting, installing updates, re-ran the command multiple times, etc., etc.. 
At that time, the only remote access I had was using Dell's iDrac, so I was trying to run the install while connected remotely using the iDrac remote viewer. I've since set up remote desktop and remote powershell so I have other methods than SSH available. SSH is still nice to have for some things, so I tried again today while connected via remote desktop. This time the install worked and I am now able to SSH in to the server. No idea if any of that makes any difference, but it's working now.
